### PR TITLE
Ut/bootstrap api

### PR DESCRIPTION
**Pull request by OrezzerO**
### Motivation:
Add some test case for RPC config classes

### CODE REVIEW

File: core/common/src/main/java/com/alipay/sofa/rpc/common/utils/TestUtils.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+ */
+package com.alipay.sofa.rpc.common.utils;
+
+import java.util.concurrent.ThreadLocalRandom;
+
+/**
+ * @author zhaowang
+ * @version : TestUtils.java, v 0.1 2022年01月25日 5:00 下午 zhaowang
+ */
+public class TestUtils {
```
Comment (JervyShi) [inline core/common/src/main/java/com/alipay/sofa/rpc/common/utils/TestUtils.java:25]: Is it necessary to put this class under the java source package?

File: core/common/src/test/java/com/alipay/sofa/rpc/common/utils/TestUtilsTest.java [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+public class TestUtilsTest {
+
+    @Test
+    public void testRandomString() {
+        Pattern compile = Pattern.compile("^[a-zA-Z0-9]+$");
+        for (int i = 0; i < 1000; i++) {
+            String s = TestUtils.randomString();
+            boolean condition = compile.matcher(s).find();
+            if (!condition) {
+                System.out.println(s);
```
Comment (JervyShi) [inline core/common/src/test/java/com/alipay/sofa/rpc/common/utils/TestUtilsTest.java:38]: Invalid test case.

**Review by JervyShi — APPROVED** [review]
LGTM

**Review by nobodyiam — APPROVED** [review]
LGTM

**Pull request merged**